src/assets/sass/application.scss

```diff
@@ -1 +1,12 @@
-@use 'govuk/all';
+@import 'govuk/all';
+
+$gov-defra-color-websafe: govuk-organisation-colour(
+  'department-for-environment-food-rural-affairs'
+);
+
+.app-main-class {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
```

Comment:
please keep all scss files in assets/sass folder itself. index.scss can import other partial scss files. rename this file as partial.

Reply:
Incorporated as per the discussion